yarn.lock

```diff
@@ -677,12 +677,6 @@
   dependencies:
     regenerator-runtime "^0.12.0"
 
-"@babel/runtime@^7.3.1":
-  version "7.4.5"
-  resolved "https://registry.yarnpkg.com/@babel/runtime/-/runtime-7.4.5.tgz#582bb531f5f9dc67d2fcb682979894f75e253f12"
-  dependencies:
```

Comment:
You'd think I'd know better by now, but `yarn.lock` changes still intimidate me.  Maybe you can enlighten me here.  Why are you checking this in if you didn't change any dependencies anywhere?

Reply:
I removed `sleep` because it wasn't being used anywhere and ran `yarn install` to update the lockfile. Possibly these are deps of `sleep`?